esp/src/eclwatch/nls/hu/hpcc.js

```diff
@@ -846,13 +867,16 @@ define(
     YouAreAboutToDeleteThisWorkunit: "Ennek a munkaegységnek a törlésére készül. Folytatja?",
     YouAreAboutToRemoveUserFrom: "Ön felhasználó(k) csoportból való törlésére készül. Kívánja folytatni?",
     YourBrowserMayNotSupport: "Előfordulhat, hogy az Ön böngészőprogramja nem támogatja az ekkora méretű fájlokat.",
+    YourScreenWasLocked: "Az ESP server befagyasztotta az ön képernyőjét. Kérem frissítsen.",
     ZAP: "Z.A.P.",
     ZeroLogicalFilesCheckFilter: "Nincs a megadott feltételnek megfelelő adat! (A jó a szűrőfeltétel?)",
-    Zip: "Zip",
+    Zip: "Irányítószám",
```

Comment:
Is this "Zip" the postcode of file extension?

Reply:
ZIP a file